db/migrate/20170717140915_add_numeric_columns_to_container_quota_items.rb

```diff
@@ -0,0 +1,44 @@
+class AddNumericColumnsToContainerQuotaItems < ActiveRecord::Migration[5.0]
+  class ContainerQuotaItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_quota_items, :quota_desired_float,  :float
+    add_column :container_quota_items, :quota_desired_int,    :bigint
```

Comment:
Sorry, I'm unconvinced.
- I could see an argument for not having any floats, only bigints, to avoid any rounding.
  But since we allow floats in *some* cases, why not use only floats?
- These come from quotas ultimately set by kubernetes users, who can express same quantity as `2048000m` cpu cores or `2Ki` cores.  The first would parse as 2048.0 float, the second as 2048 int, but I don't see why keep the distinction to any code that will deal with these columns...
- AFAIK `m`illi is the only sub-integer unit in this use case, so if we really wanted no rounding we could store everything in 1000x scaled fixed point.  Ugly but better than 2 columns IMHO.

P.S. this may be out of scope, but I wish we stored units somewhere, and not just inferred them from `resource`.  does quota_enforced_int == 1000 mean 1kBytes ? 1kdollars ? 1ksievert ?